pkgs/applications/networking/browsers/google-chrome/default.nix

```diff
@@ -119,12 +119,6 @@ in stdenv.mkDerivation {
     cp -a opt/* $out/share
     cp -a usr/share/* $out/share
 
-    # To fix --use-gl=egl:
-    test -e $out/share/google/$appname/libEGL.so
-    ln -s libEGL.so $out/share/google/$appname/libEGL.so.1
```

Comment:
Oh, I should mention to document this here: `libEGL.so.1` can (and likely must) be dropped as it will be used from `libglvnd`. I thought this was a wrapper for ANGLE but either that isn't the case or the wrapper's compatible with libEGL from the system and can be replaced by it.

Anyway, the rest of those files don't seem to get used (at least on my system they weren't - according to strace).